app/templates/components/component-tree-item.hbs

```diff
@@ -21,14 +20,46 @@
   {{/if}}
 
   <code
-    class="component-tree-item__tag {{if @item.isComponent "component-tree-item__bracket"}} {{if (not @item.hasChildren) "component-tree-item__bracket--self-closing"}}"
+    class="component-tree-item__tag
+      {{if
+        (and (not @item.args.positional) @item.isComponent)
```

Comment:
We use `@item.args.positional` a lot so let's make it a getter in the class, i.e. `isCurlyInvocation` (in that case we can fold `@item.isComponent` into it also. The added benefit is that if we have a better way to detect curly invocation later we can just change that one place.